src/pygambit/game.pxi

```diff
@@ -21,6 +21,7 @@
 #
 import itertools
 import pathlib
+import threading
```

Comment:
There is no need to introduce threading (and threading.Lock doesn't do what you seem to think it does anyway).